package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "cov-clear-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest src",
+    "test:watch": "jest src --watch",
+    "develop": "nodemon",
+    "build": "tsc",
+    "start": "node dist",
+    "format": "prettier --write --single-quote src"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.ts": [
+      "npm run format"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cov-clear/backend.git"
+  },
+  "author": "",
+  "license": "UNLICENSED",
```

Comment:
Apache?

Reply:
Will add yep, left it as a todo until we decide.